extensions/sql-migration/src/api/sqlUtils.ts

```diff
@@ -49,7 +49,7 @@ const query_databases_with_size = `
 		db_size
 		AS
 		(
-			SELECT database_id, CAST(SUM(size) * 8.0 / 1024 AS INTEGER) size
+			SELECT database_id, CAST(SUM(size) / 128 AS bigint) size
```

Comment:
this query was taken from the ADS implementation and simplified...  Did the original query address this issue?  Does it need to be fixed as well?

Reply:
Original query has the issue [OriginalQuery](https://github.com/microsoft/sqltoolsservice/blob/f288bee29418fd205a9d14e3c3a4cac360caa11d/src/Microsoft.SqlTools.ServiceLayer/Connection/ListDatabaseRequestHandler.cs#L174.) , we need to communicate to the SQL tools team, not sure if we have an internal channel or just create an issue on ADS.

Reply:
@junierch just wondering what is the size of your test database. the max value for int is 2,147,483,647, the cast happens at the end of the type casting, so IIUC, the database size must be more than 2 PB in order to hit the issue.

Reply:
I picked the next exact-number data type to prevent conversion issues.